### PR TITLE
Longitude and latitude fields land in proper data roles by default

### PR DESCRIPTION
**Pull request by baspeti**

### CODE REVIEW

**Comment by ryanbaumann** [comment]
Nice find @baspeti!  Merge away if this works 👍 .

**Comment by baspeti** [comment]
Unfortunately it seems that currently Microsoft BI attempts to perform some sort of aggregation function on the Latitude/Longitude data fields, even if I work with a Power BI data set, where I explicitly set Latitude and Longitude columns to types of latitude and longitude respectively in Power BI desktop, and not to summarize them by default.

image

And I believe it is taking it into account that I explicitly told the type for Latitude and Longitude, because if I work with the data set exported from Power BI desktop, the Latitude and Longitude data fields have a little globe icon, instead of the summa icon:

image

But even in this case, only by ticking into the checkbox next to them, the resulting data roles will be average values:

image

**Comment by baspeti** [comment]
And @ryanbaumann , I guess you were asking why we changed `Grouping` to `GroupingOrMeasure` for longitude and latitude data fields, right? As I mentioned in my previous comment that Power BI executes Average aggregation by default, no matter what you chose in Power BI desktop as `Default Summarization` on Latitude and Longitude fields, the target data role has to be `Measure` or `GroupingOrMeasure`. Actually it was not working before, because their types were `Grouping`, and I guess Power BI simply skipped them as possible default data role targets, but I don't know how Power BI works deep inside.

The bottom line, for the current achievement the key was *not* to add `preferredTypes` for latitude and longitude roles. I believe it will only be useful in the future.

@uwe confirmed that this default summarization is known bug, even if you selected `Don't summarize`, but not clear when Microsoft will resolve it.
https://github.com/Microsoft/PowerBI-visuals/issues/297#issuecomment-360488857

**Comment by ryanbaumann** [comment]
OK @baspeti, given Microsoft/PowerBI-visuals#297 (comment), let's merge as-is 👍
